Client/Frontend/Browser/SearchViewController.swift

```diff
@@ -728,6 +783,19 @@ class SearchViewController: SiteTableViewController,
             twoLineCell.leftImageView.setFavicon(FaviconImageViewModel(urlStringRequest: site.url))
             twoLineCell.accessoryView = nil
             cell = twoLineCell
+        case .fakeTestSection:
+            let suggestion = fakeTestSuggestions[indexPath.row]
+            oneLineCell.titleLabel.text = suggestion.title
+            oneLineCell.leftImageView.contentMode = .center
```

Comment:
This line is what is causing the zoomed in favicon issue. A content mode of type center means it will use the full size of the raw image and it will center it on the view, meaning if the image is bigger than the view it will be partially cut off. 
By default for the favicon view we use scaleAspectFit, scaleAspectFill would also work because the images are square. These content modes ensure the images get scaled down to fit the view while maintaining the images aspect ratio.
Since these are set by the favicon view automatically you can just remove this line and it looks correct.